pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/creator/TransformPipelineTest.java

```diff
@@ -25,18 +25,16 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-import static org.apache.pinot.core.data.manager.realtime.Fixtures.*;
```

Comment:
removing this static import doesn't improve this class

Reply:
I think this is because originally I did not use `Fixtures.createSchema()` in code?
After the refactor the IntelliJ editor tries to remove this

Reply:
I'm not too keen on automatic IDE actions, this and others are completely unrelated to the change in this PR which will clutter up the git history

Reply:
fair enough. Changed now

Reply:
I'd suggest removing the static import especially on function usage though